src/Popover/Popover.tsx

```diff
@@ -60,7 +60,7 @@ export namespace Popover {
   export type Delay = number | { whenClosed?: number; whenOpen?: number };
 
   export type PopoverSettings = {
-    content: React.ReactNode;
+    content: NonNullable<Children>;
```

Comment:
it doesn't make any sense to render an empty tooltip right? (this still allows for `false` to be passed here... anyway)